derive/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+mod js_enum;
+
+use proc_macro::TokenStream;
+
+#[proc_macro_derive(JsEnum)]
+pub fn js_enum(input: TokenStream) -> TokenStream {
```

Comment:
Why is this required? Can't re-export?

Reply:
no, proc_macros must be defined in the top-level of a crate :(

Reply:
agh, sucks